### ISSUE TITLE
Can mix ggPMX and ggplot syntax safely?

### ISSUE THREAD

**Issue by KZRKZR** [ggPMXdevelopment/ggPMX#84]
Can we trust the following syntax:

`ctr %>% pmx_plot_npde_pred()+facet_wrap(~SEX)`
will be result the same with:
`ctr %>% pmx_plot_npde_pred(strat.facet= ~SEX)`

First impression  - probably (see the plot_pmx methods)
Some error handling may be necessary, or documentation

**Comment by tynsci** [comment]
Good point @KZRKZR :+1: 
- *Yes*, the ggPMX plotting functions return objects which inherit "gg" and "ggplot"
  - this means that you can keep adding components to it using [the standard `ggplot2` syntax](https://ggplot2.tidyverse.org/reference/gg-add.html)
  - documenting all the possibilities and possible errors is not feasbible
- **Suggest** noting this explicitly in the README

**Comment by baltcir1** [comment]
I agree. From my experience, no issues in mixing ggplot syntax with ggPMX output.
